resources/views/admin/courses/index.blade.php

```diff
@@ -0,0 +1,46 @@
+@extends('admin.layouts.master')
+@section('content')
+    <div class="row">
+        <div class="col-xl-12 col-lg-12 col-md-12 col-sm-12 col-12">
+            <div class="page-header">
+                <h2 class="pageheader-title">{{ trans('list_user.title') }}</h2>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+        <!-- ============================================================== -->
+        <!-- basic table  -->
+        <!-- ============================================================== -->
+        <div class="col-xl-12 col-lg-12 col-md-12 col-sm-12 col-12">
+            <div class="card">
+                <div class="card-body">
+                    <div class="table-responsive">
+                        <table class="table table-striped table-bordered first">
+                            <thead>
+                            <tr>
+                                <th>{{ trans('list_courses.table.name') }}</th>
+                                <th>{{ trans('list_courses.table.description') }}</th>
+                                <th>{{ trans('list_courses.table.category') }}</th>
+                                <th>{{ trans('list_courses.table.price') }}</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            @foreach($courses as $course)
+                                <tr>
+                                    <td>{{ isset($course->name) ? $course->name : '' }}</td>
```

Comment:
nếu không tồn tại thì in ra làm chi..
ví dụ tất cả các biêns nớ ko tồn tại thì trả về trang rỗng

Reply:
ai bao m k tồn tại trả vê rỗng, ?????

Reply:
nếu m k isset chổ này nó sẻ lỗi nêu data null

Reply:
giờ mà ko có khóa học nào hết thì hắn sẽ hiển thị cái gì?
có báo lỗi ko?

Reply:
check rồi thi lỗi gì nữa.